src/components/SummarySection.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
+import Go123Box from "./Go123Box";
+import SummaryBox from "./SummaryBox";
+
+const SummarySection = ({ className, gridClassName, summaryList }) => {
+  return (
+    <div className={className}>
+      <h2 className="pb-10 uppercase text-h3">The Summary</h2>
+      <div className={`grid ${gridClassName}`}>
+        {summaryList.map((item, i) =>
+          Object.prototype.hasOwnProperty.call(item, "quantity") ? (
```

Comment:
`Object.prototype.hasOwnProperty.call(item, "quantity")` should just be `item.quantity`, or if you want to be more correct in accounting for the 0 case then `item.quantity != null`